### PR TITLE
Reflected BLE iBeacon changes made by home-assistant/android/pull/1856

### PR DESCRIPTION
**Pull request by amadeo-alex**
Added information in regards to changes proposed by https://github.com/home-assistant/android/pull/1856
(additional setting for advertise mode selection)

### CODE REVIEW

File: docs/core/sensors.md [hunk 1]
```diff
@@ -242,7 +242,11 @@ A BLE Transmitter sensor allows your device to transmit a BLE iBeacon.  This is
...
+Settings are available to change the UUID, Major and Minor masks. These can be used to change the overall identifier, as well as to allow groups, e.g. family phone devices can have particular Major value which can be whitelisted in apps like roomassistant. These settings are validated: UUID should be the [standard format](https://en.wikipedia.org/wiki/Universally_unique_identifier), Major and Minor need to be within 0 and 65535. 
+There are also settings to alter:
+*   the Transmit power (between Ultra Low, Low, Medium and High)
+*   the Advertise mode (between Low Power (1Hz), Balanced (3Hz) and Low latency (10Hz))
```
Comment (dshokouhi) [inline docs/core/sensors.md:248]: Please add a beta flag for this as its new in the beta only, later on when we cut a release all beta flags will then be removed

**Review by dshokouhi — CHANGES_REQUESTED** [review]
We should make sure to be explicit about what features are in the beta so users do not get confused :)

**Comment by amadeo-alex** [comment]
You're right, I should've thought about it :D
I've browsed through the docs sources and deducted that by BETA you meant the "<span class="beta">BETA</span>" - correct me please if I'm wrong.

**Review by dshokouhi — APPROVED** [review]
Looks good to me, can be merged once the parent is merged.

**Pull request merged**